docs/en/operations/table_engines/mergetree.md

```diff
@@ -313,6 +313,17 @@ INDEX sample_index2 (u64 * length(str), i32 + f64 * 100, date, str) TYPE set(100
 INDEX sample_index3 (lower(str), str) TYPE ngrambf_v1(3, 256, 2, 0) GRANULARITY 4
 ```
 
+#### Functions Support
+
+
+Comparison and search functions can use different indexes instead of the full scan of the table during the query processing. All the functions can use primary key index. Some of the functions can use data skipping indexes.
+
+ Index | Functions
+-------|-----------
+`ngrambf_v1` | [equals](../../query_language/functions/comparison_functions.md#function-equals), [notEquals](../../query_language/functions/comparison_functions.md#function-notequals), [like](../../query_language/functions/string_search_functions.md#function-like), [notLike](../../query_language/functions/string_search_functions.md#function-notlike), [startsWith](../../query_language/functions/string_functions.md#function-startswith), [endsWith](../../query_language/functions/string_functions.md#function-endswith), [multiSearchAny](../../query_language/functions/string_search_functions.md#function-multisearchany), [in](../../query_language/functions/in_functions.md#in-functions), [notIn](../../query_language/functions/in_functions.md#in-functions)
```

Comment:
1. We'd better include human readable operator names like == or != as well
2. I'd transpose this to be a table like this one https://clickhouse.yandex/docs/en/interfaces/formats/ with index types horizontally and function names vertically
3. Some footnotes mentioning limitations would be useful, for example will the index be used if startsWith looks for a prefix smaller than ngram size?

Reply:
Notes about 3

- Functions with a constant argument less than ngram size couldn't be used by ngrambf_v1 for query optimization.

- Bloom filters can have false positive matches, so ngrambf_v1 couldn't optimize queries where result of a function expected to be false.
      Examples:
      **Can be optimized:**
            s LIKE '%test%'
            NOT s  NOT LIKE '%test%'
            s = 1
            NOT s != 1
            startsWith(s, 'test')
      **Can't be optimized:**
            NOT s LIKE '%test%'
            s  NOT LIKE '%test%'
            NOT s = 1
            s != 1
            NOT startsWith(s, 'test')